components/popover/index.css

```diff
@@ -150,7 +150,8 @@ governing permissions and limitations under the License.
 	/* spacing to include tip in calculation of offset from source */
 	&.spectrum-Popover--withTip {
 		/* tip size minus where it overlaps with popover border */
-		margin-inline-start: calc(var(--mod-popover-pointer-width, var(--spectrum-popover-pointer-width)) - var(--mod-popover-border-width, var(--spectrum-popover-border-width)));
+		/* stylelint-disable-next-line csstools/use-logical -- intentional, right stays on the same side even for RTL */
```

Comment:
Thanks for adding this detailed comment!  Very helpful and ensures we don't accidentally "correct" it later.